docker/docker-compose.yml

```diff
@@ -0,0 +1,27 @@
+version: '3'
+
+services:
+  frontend:
+    build:
+      context: ..
+      dockerfile: docker/frontend.dockerfile
+    image: frontend
+    stdin_open: true
```

Comment:
I don't mind that this is here but why do we need/want this?? It should probably be documented.
Or is this supposed to be on the backend container for python debugging??

Reply:
stdin_open is required to run the node server in development, otherwise it crashes before the server starts